project/MimaExcludes.scala

```diff
@@ -216,14 +216,6 @@ object MimaExcludes {
     ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.ml.regression.RandomForestRegressionModel.setMinInstancesPerNode"),
     ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.ml.regression.RandomForestRegressionModel.setNumTrees"),
 
-    // [SPARK-26124] Update plugins, including MiMa
```

Comment:
@huaxingao, can you point out why this is not needed anymore?

Reply:
I think these are not needed any more because of the following was added in https://github.com/apache/spark/pull/23086
```
    // Data Source V2 API changes
    (problem: Problem) => problem match {
      case MissingClassProblem(cls) =>
        !cls.fullName.startsWith("org.apache.spark.sql.sources.v2")
      case MissingTypesProblem(newCls, _) =>
        !newCls.fullName.startsWith("org.apache.spark.sql.sources.v2")
      case InheritedNewAbstractMethodProblem(cls, _) =>
        !cls.fullName.startsWith("org.apache.spark.sql.sources.v2")
      case DirectMissingMethodProblem(meth) =>
        !meth.owner.fullName.startsWith("org.apache.spark.sql.sources.v2")
      case ReversedMissingMethodProblem(meth) =>
        !meth.owner.fullName.startsWith("org.apache.spark.sql.sources.v2")
      case _ => true
    }
```
Then, because of https://github.com/apache/spark/pull/25700
```
org.apache.spark.sql.sources.v2.reader -> org.apache.spark.sql.connector.read
```
I think we can actually change the above code to the following
```
    (problem: Problem) => problem match {
      case MissingClassProblem(cls) =>
        !cls.fullName.startsWith("org.apache.spark.sql.sources.v2")
      case _ => true
```